tiri/src/lib.rs

```diff
@@ -45,12 +42,13 @@ use yktrace::tir::{
 #[allow(dead_code)]
 const DUMP_CONTEXT: usize = 4;
 
-/// Each local variable is represented by one `Value` struct.
+/// The contents of each local variable is represented by a `Value`.
 #[derive(Debug, Clone, Eq, PartialEq)]
 enum Value {
-    /// A stand-alone immediate value.
+    /// A value that can be stored immediately into a local variable without touching memory.
     Immediate(Immediate),
-    /// Memory.
+    /// A reference to a potentially larger (and potentially composite) value in memory (either on
```

Comment:
"A `Value` which is too big for us to want to store in register(s)." Note that I'm assuming that there are some values which are, say, 6 machine words big and which we *could* store in registers, but won't, because it's not a very good use of them.